apps/dashboard/src/components/workflow-editor/schema.ts

```diff
@@ -94,3 +94,25 @@ export const makeObjectFromSchema = ({
     return { ...acc, [key]: value.default };
   }, {});
 };
+
+const ChannelPreferenceSchema = z.object({
```

Comment:
Do we maybe have these schemas in the API code so that we can reuse them?

Reply:
I didnt find the schemas, there were dtos and types only